Makefile

```diff
@@ -111,7 +126,7 @@ docs/%.8: docs/%.8.md check-gopath
 
 docs: $(MANPAGES)
 
-install: check-gopath
+install: ocid ocic kpod conmon/conmon pause/pause
```

Comment:
i've remove the files from the recipe, but also the check-gopath, because it is not needed at this point either.